gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemImpl.java

```diff
@@ -124,12 +145,7 @@ public class GoogleCloudStorageFileSystemImpl implements GoogleCloudStorageFileS
    */
   public GoogleCloudStorageFileSystemImpl(
       Credentials credentials, GoogleCloudStorageFileSystemOptions options) throws IOException {
-    this(
-        GoogleCloudStorageImpl.builder()
-            .setOptions(checkNotNull(options, "options must not be null").getCloudStorageOptions())
-            .setCredentials(credentials)
-            .build(),
-        options);
+    this(createCloudStorage(options, credentials, null), options);
```

Comment:
nit: add comment to `null` param: `/* downscopedAccessTokenFn= */`

Reply:
Done.